packages/api/src/controllers/EpochController.js

```diff
@@ -14,9 +14,10 @@ class EpochController {
 
   getEpochByIndex = async (request, response) => {
     const { index } = request.params
+    const { ascending = true } = request.query
```

Comment:
What this param do? We should mention in the docs

Reply:
prev/next epoch info

Reply:
✅